plugins/baseball-scoreboard/game_renderer.py

```
@@ -165,13 +165,16 @@ def _load_and_resize_logo(self, league: str, team_abbrev: str) -> Optional[Image
                 if logo.mode != 'RGBA':
                     logo = logo.convert('RGBA')
 
-                # Crop transparent padding then scale so ink fills display_height.
-                # thumbnail into a display_height square box preserves aspect ratio
-                # and prevents wide logos from exceeding their half-card slot.
+                # Crop transparent padding so scaling operates on actual content.
+                # Then constrain to the logo slot width and 75% of display height —
+                # this prevents wide 1200×630 source images (common in MiLB/ESPN)
```

Ruff (0.15.15)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)